### ISSUE TITLE
fix recipes

### ISSUE THREAD

**Issue by barti2019** [nicholasgower/planet-muluna#16]
tell me, how to reach on that muluna without discover any technology needed to travel in space. I cannot start game on the moon and i cannot  research any nnecessary technology to travel in space without getting there. change that its problematic

**Comment by nicholasgower** [comment]
I'm not sure what you mean. On the latest version of the mod, starting with rocket silo researched, I was able to reach Muluna without cheating in any technologies.

**Comment by barti2019** [comment]
i had turned off all mods. only pack ,,all planets,, was installed and even there was problem with research crushers and recipes on them and thrusters. You need space science packs to unlock them or stand on Muluna and without space technology its imposible to unlock

**Comment by nicholasgower** [comment]
Can you send me your save where you're having this issue?

**Comment by nicholasgower** [comment]
Also, if you aren't aware, you can get to Muluna without crushers or asteroid crushers if you send barrels of water, iron ore, and rocket fuel up to your ship.

**Comment by nicholasgower** [comment]
A ship like this one is capable of getting to Muluna.

![image](https://github.com/user-attachments/assets/fd80c2e9-e34c-44df-8f80-9d7fe70fb27a)

**Comment by barti2019** [comment]
I never think about sending any of that resources in this stage of game, propably im noob. I will try that later. 
But the mods like ,,planet picker,, in this solution is much much harder to play but not imposible, maby u would make similar moons for the rest of basic planets. Its good mod and give some moore oportunity  on start.
Thanks very much for advice, you change my mind in this problem, in next sesion in game i would try that.

**Comment by nicholasgower** [comment]
Yeah, I'm aware that Planet picker would be difficult to make compatible with this mod without moving Muluna to wherever the starting planet is. I think this is more a job for the creator of Planet Picker to deal with. You're welcome to ask them to make their mod more compatible with mine.

I'm happy that I was able to help you get through the initial hurdle of this mod. Hope you enjoy it. I've updated this mod's description to make the early game's expectations clearer.
